pkg/plugins/platform/local.go

```diff
@@ -281,9 +279,34 @@ func (l *LocalApp) DockerComposeYAMLPath() string {
 	return l.AppConfig.DockerComposeYAMLPath()
 }
 
+// ComposeFiles returns a list of compose files for a project.
+func (l *LocalApp) ComposeFiles() []string {
+	files, err := util.FindFiles(path.Join(l.AppRoot(), ".ddev"), "docker-compose")
```

Comment:
What about moving the docker compose files into their own dir under .ddev, where they're the only legitimate occupants?